beta/src/components/Layout/Sidebar/Sidebar.tsx

```diff
@@ -13,32 +13,44 @@ import {MobileNav} from '../Nav/MobileNav';
 import {Feedback} from '../Feedback';
 
 const SIDEBAR_BREAKPOINT = 1023;
+const SM_BREAKPOINT = 640;
 
 export function Sidebar() {
   const {menuRef, isOpen} = React.useContext(MenuContext);
   const isMobileSidebar = useMediaQuery(SIDEBAR_BREAKPOINT);
   let routeTree = React.useContext(SidebarContext);
   const isHidden = isMobileSidebar ? !isOpen : false;
 
+  const bannerHeight = useMediaQuery(SM_BREAKPOINT) ? 72 : 40;
+  let navMarginTop = bannerHeight;
+  if (typeof window !== 'undefined') {
```

Comment:
Different render outputs for server and initial client output means hydration can produce mismatches. This doesn't seem right.